dynamodb/data.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dynamodb
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
+)
+
+// ProcessData performs data conversion for DynamoDB database. For each table,
+// we extract data using Scan requests, convert the data to Spanner data (based
+// on the source and Spanner schemas), and write it to Spanner. If we can't
+// get/process data for a table, we skip that table and process the remaining
+// tables.
+func ProcessData(conv *internal.Conv, client dynamoClient, consistentRead bool) error {
+	for srcTable, srcSchema := range conv.SrcSchema {
```

Comment:
Quick question: we should change postgres/infoschema.go ProcessSQLData to use this same iteration structure https://github.com/cloudspannerecosystem/harbourbridge/blob/master/postgres/infoschema.go#L75. What do you think?
Not suggesting we should to it in this CL.

Reply:
Yeah, this sounds good to me.